### PR TITLE
add libreplanet 2020 event

### PR DESCRIPTION
**Pull request by ian-kelling**

### CODE REVIEW

File: menu/libreplanet_2020.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "start": "2019-03-14",
+    "end": "2019-03-15",
+    "version": 2020021000,
+    "url": "https://libreplanet.org/2020/lp2020-schedule.ics",
+    "id": "LibrePlanet 2020",
```
Comment (Wilm0r) [inline menu/libreplanet_2020.json]: This is no longer needed.

**Comment by Wilm0r** [comment]
https://travis-ci.org/Wilm0r/giggity/builds/648694991?utm_source=github_status&utm_medium=notification

**Comment by ian-kelling** [comment]
Removed id. Take another look.

**Comment by Wilm0r** [comment]
Travis checks still not passing..

**Comment by mmuman** [comment]
"inconsistent indentation": you'll want to use only tabs or only spaces instead of mixing them.

**Comment by Wilm0r** [comment]
Feel free to reopen once you discover how to click on the Travis link...

**Pull request closed without merging**